src/components/dev-hub/card-list.js

```diff
@@ -65,29 +65,52 @@ const renderPodcast = podcast => (
 );
 
 const renderContentTypeCard = item => {
-    if (item.mediaType)
+    if (item.mediaType) {
         switch (item.mediaType) {
             case 'youtube':
             case 'twitch':
                 return renderVideo(item);
             case 'podcast':
                 return renderPodcast(item);
             default:
-                return;
+                return renderArticle(item);
         }
+    }
+    // Some items don't have mediaType. In that case, they are articles
```

Comment:
Do we hit line 80 anymore? Is there any case where we don't assign `mediaType`?

Reply:
Yes, initially mediaType was not assigned `SearchArticleResult` class, so I added it in. However, I was not too sure if mediaType was not assigned in some other logic